File: tests/testdata/codegen/random-pet-pp/go/random-pet.go
package main

import (
	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := random.NewRandomPet(ctx, "random-pet", &random.RandomPetArgs{
			Prefix: "doggo",
		})
		if err != nil {
			return err
		}
		return nil
	})
}

File: pkg/codegen/testing/utils/testdata/random-pet-pp/dotnet/random-pet.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Random = Pulumi.Random;

return await Deployment.RunAsync(() => 
{
    var random_pet = new Random.RandomPet("random-pet", new()
    {
        Prefix = "doggo",
    });

});

